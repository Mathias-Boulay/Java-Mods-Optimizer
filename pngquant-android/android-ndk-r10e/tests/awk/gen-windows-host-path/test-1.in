C:/cygwin/bin on /usr/bin type ntfs (binary,auto)
C:/cygwin/lib on /usr/lib type ntfs (binary,auto)
C:/cygwin on / type ntfs (binary,auto)
C: on /cygdrive/c type ntfs (binary,posix=0,user,noumount,auto)
D: on /cygdrive/d type udf (binary,posix=0,user,noumount,auto)
