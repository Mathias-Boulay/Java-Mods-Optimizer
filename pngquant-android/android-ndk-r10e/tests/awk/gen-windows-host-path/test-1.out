$(patsubst /%,C:/cygwin/%,
$(patsubst /usr/bin/%,C:/cygwin/bin/%,
$(patsubst /usr/lib/%,C:/cygwin/lib/%,
$(patsubst /cygdrive/C/%,C:/%,
$(patsubst /cygdrive/D/%,D:/%,
$(patsubst /cygdrive/c/%,C:/%,
$(patsubst /cygdrive/d/%,D:/%,
$1)))))))
